cmd/pitr/recoverer/recoverer.go

```diff
@@ -272,6 +294,47 @@ func getLastBackupGTID(infoObj io.Reader) (string, error) {
 	return string(set), nil
 }
 
+func getDecompressedContent(infoObj io.Reader, format string) ([]byte, error) {
+	tmpDir := os.TempDir()
+	fileName := "backup_info." + format
+	tmpFile, err := os.Create(tmpDir + "/" + fileName)
+	if err != nil {
+		return nil, errors.Wrap(err, "create temp info file")
+	}
+	defer func() {
+		tmpFile.Close()
+	}()
+
+	content, err := ioutil.ReadAll(infoObj)
+	if err != nil {
+		return nil, errors.Wrap(err, "read info object")
+	}
+	_, err = tmpFile.Write(content)
+	if err != nil {
+		return nil, errors.Wrap(err, "write content to temp file")
+	}
+
+	cmdString := "cd " + tmpDir + " && xbstream -x --decompress < " + fileName
```

Comment:
Do we really need to uncopress it with `xbstream`?
Can we use go libraries instead?

Reply:
We already have xbstream inside image and it works with xtrabackup files and all possible compressions